bin/create-secrets.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+OPENSTACK_CREDENTIALS_PATH=$1
+SPI_CREDENTIALS_PATH=$2
+
+get_secret_object(){
+    local NAME=$1
+    local DATA=$2
+    cat <<EOF
+{
+  "kind": "Secret",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "$NAME"
+  },
+  "data": {
+     $DATA
+  }
+}
+EOF
+}
+
+get_data_from_path(){
+    cat $1 | base64 | awk 'BEGIN{ORS="";} {print}'
+}
+
+OPENSTACK_CREDENTIALS_SECRET=$(get_secret_object novarc \
+                                                 "\"novarc\": \"$(get_data_from_path $OPENSTACK_CREDENTIALS_PATH)\"")
+SPI_CREDENTIALS_SECRET=$(get_secret_object spi.ini \
+                                           "\"spi.ini\": \"$(get_data_from_path $SPI_CREDENTIALS_PATH)\"")
```

Comment:
these lines have weird identation. Shouldn't they have only two spaces from the start of the line?